_includes/head-custom.html

```diff
@@ -0,0 +1 @@
+<link rel="shortcut icon" type="image/x-icon" href="../images/pspdev.ico">
```

Comment:
I think the problem is not where the code is, but the path that is set. I think it should be this:
```suggestion
<link rel="shortcut icon" type="image/x-icon" href="images/pspdev.ico">
```
Right now I'm seeing a 404 in Firefox as it tries to access https://pspdev.github.io/images/pspdev.ico instead of 	https://pspdev.github.io/psplinkusb/images/pspdev.ico